Add files via upload

thakral-kartik opened the pull request:
Affiliation changed for Dr. Mayank Vatsa and Dr. Richa Singh from IIIT Delhi to IIT Jodhpur and updated their websites.

# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

_Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

**Inclusion criteria**

- [ ] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [ ] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [ ] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

thakral-kartik committed:
Add files via upload

Affiliation changed for Dr. Mayank Vatsa and Dr. Richa Singh from IIIT Delhi to IIT Jodhpur and updated their websites.

emeryberger commented on csrankings.csv:
```diff
@@ -12330,7 +12330,7 @@ Maya Ramanath,IIT Delhi,http://www.cse.iitd.ac.in/~ramanath,y6BXzxIAAAAJ
 Mayank Goel,Carnegie Mellon University,https://www.hcii.cmu.edu/people/mayank-goel,32eGduMAAAAJ
 Mayank Singh 0001,IIT Gandhinagar,http://www.iitgn.ac.in/faculty/comp/mayank.htm,FOjZrQcAAAAJ
-Mayank Vatsa,IIIT Delhi,https://www.iiitd.edu.in/~mayank,NOSCHOLARPAGE
+Mayank Vatsa, IIT Jodhpur, http://home.iitj.ac.in/~mvatsa/, -DAxp-cAAAAJ&hl
```
Google Scholar name is wrong (spurious `&hl`).

thakral-kartik committed:
Resolving affiliation of Dr. Mayank Vatsa and Dr.Richa Singh and adding new faculties to the list

Changing the affiliation of Dr. Mayank Vatsa and Dr. Richa Singh from IIIT Delhi to IIT Jodhpur and Dr. Anand Mishra and Dr Yashaswi Verma to the csv file.

thakral-kartik commented:
Creating a new pull request with updated csv
